package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "angular-loading-bar",
-  "version": "0.9.0",
+  "version": "0.10.0",
```

Comment:
Please revert this version bump, if the PR is merged we'll bump the version as part of the release process (which will generate a git tag as well)